src/adafruit_blinka/microcontroller/mcp2221/pin.py

```diff
@@ -2,7 +2,10 @@
 #
 # SPDX-License-Identifier: MIT
 """MCP2221 pin names"""
-from .mcp2221 import mcp2221
+from .mcp2221 import MCP2221
+
+
+mcp2221 = MCP2221()
```

Comment:
The original import should still work, but I prefer not to rely on the global `mcp2221` var.

Reply:
I'm going to revert this change, since it causes the opening of two different MCP2221 if the first MCP2221 is not available, as @fgervais discovered.